rtlib/include/rt/LandmarkIO.hpp

```diff
@@ -16,25 +17,16 @@ class LandmarkIO {
 
     void setFixedImage(const Image8UC3::Pointer i) { fixedImage_ = i; }
     void setMovingImage(const Image8UC3::Pointer i) { movingImage_ = i; }
-    void setLandmarksPath(const boost::filesystem::path& path)
-    {
-        landmarksPath_ = path;
-    }
+    void setLandmarksPath(const boost::filesystem::path& path) { landmarksPath_ = path; }
```

Comment:
Did you run clang-format on this? I'm pretty sure I did and it was over the 80 character line limit.